rotkehlchen/db/history_events.py

```diff
@@ -500,10 +502,18 @@ def get_history_events(
                     deserialized_event = EthDepositEvent.deserialize_from_db(data)
 
                 else:
-                    data = entry[data_start_idx:HISTORY_BASE_ENTRY_LENGTH + 1]
-                    deserialized_event = HistoryEvent.deserialize_from_db(entry[data_start_idx:])
+                    data = entry[data_start_idx:]
+                    deserialized_event = HistoryEvent.deserialize_from_db(data)
             except (DeserializationError, UnknownAsset) as e:
-                log.debug(f'Failed to deserialize history event {entry} due to {e!s}')
+                error = f'Failed to deserialize history event {entry} due to {e!s}'
+                log.error(error)
```

Comment:
why create a variable if it is only used here? 

```suggestion
                log.error(f'Failed to deserialize history event {entry} due to {e!s}')
```